examples/postgres/verify.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -e
+
+export NAME=postgres
+export DELAY=10
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+_postgres () {
+    local postgres_client
+    postgres_client='docker run -i --rm --network envoymesh -e PGSSLMODE=disable -e PGPASSWORD=postgres postgres:latest psql -U postgres -h envoy -p 1999'
+    ${postgres_client} <<EOF
+        ${1}
+EOF
+}
+
+run_log "Create a postgres database"
+_postgres "CREATE DATABASE test;"
+_postgres "\l" | grep test
+
+run_log "Create a postgres table"
+_postgres <<EOF
+\c test
+CREATE TABLE test ( f VARCHAR );
```

Comment:
ideally, i think map the commands in the docs step-by-step

Reply:
Make total sense. Fixed!